library/StandardLibrary/test/Sexp/SimplifiedPasses.hs

```diff
@@ -0,0 +1,353 @@
+module Sexp.SimplifiedPasses where
+
+import qualified Data.Set as Set
+import Juvix.Library
+import qualified Juvix.Library.Sexp as Sexp
+import qualified Test.Tasty as T
+import qualified Test.Tasty.HUnit as T
+import Prelude (error)
+
+--------------------------------------------------------------------------------
+-- Exported Tests
+--------------------------------------------------------------------------------
+
+top :: T.TestTree
+top =
+  T.testGroup
+    "sexp simplified pass tests:"
+    [ condWorksAsExpected,
+      ifWorksAsExpected,
+      letWorksAsExpected,
+      doWorksAsExpected,
+      recordWorksAsExpected,
+      sigandDefunWorksAsExpetcted,
+      moduleExpandsAsExpected
+    ]
+
+--------------------------------------------------------------------------------
+-- Simple Passes
+--------------------------------------------------------------------------------
+
+moduleTransform :: Sexp.T -> Sexp.T
```

Comment:
Why does this exist in both the test file and the `Passes` file, shouldn't we be testing that one?

Reply:
Simplified passes do not change with time, and are the simplified version of many pass types. I guess we can view it as a verified protoype. They also will keep passes we will remove from the frontend language etc etc.